doc/environment-rtd.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - jupyterlab==3.0.16
   - laspy==1.7.0
   - python-pdal==2.4.0
+  - gdal == 3.3.0
```

Comment:
Super minor nitpick: Use `package==version`, not `package  == version` for consistency.